docs/contributions/README.md

```diff
@@ -0,0 +1,73 @@
+# How to Contribute
+
```

Comment:
I would include a general link to our Code of Conduct

https://github.com/material-components/material-components/blob/develop/CODE_OF_CONDUCT.md